File: CTDK_Python/076/_076.py
x = float(input("Nhap x: "))
n = int(input("Nhap n: "))

s = 1 + x
t = x
m = 1
i = 3
while(i <= 2 * n + 1):
    t = t * x * x
    m = m * i * (i - 1)
    s = s + t / m
    i = i + 2

print("Tong la: ", s)

File: CTDK/076/076.cpp
#include <iostream>
using namespace std;

int main()
{
	float x;
	cout << "Nhap x: ";
	cin >> x;

	int n;
	cout << "Nhap n: ";
	cin >> n;

	float s = 1 + x;
	float t = x;
	int m = 1;
	int i = 3;
	while (i <= 2 * n)
	{
		t = t * x * x;
		m = m * (i - 1) * i;
		s = s + t / m;
		i = i + 2;
	}
	cout << "Tong la: " << s << endl;
	return 0;
}